### PR TITLE
[12.0][MIG] account invoice currency

### PR DESCRIPTION
**Pull request by yweng8111**

### CODE REVIEW

**Comment by tbaden** [comment]
runbot warning:
Two fields (amount_tax_signed, amount_tax) of account.invoice() have the same label: Tax.

**Comment by MiquelRForgeFlow** [comment]
@yweng8111 could you fix this?

File: account_invoice_currency/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+{
+    'name': "Company currency in invoices",
+    'version': "12.0.1.0.0",
+    'author': "Zikzakmedia SL, "
+              "Joaquín Gutierrez, "
+              "Tecnativa, "
+              "Softdil S.L, "
+              "Odoo Community Association (OCA) ",
+    'website': "https://odoo-community.org/",
```
Comment (nikul-serpentcs) [inline account_invoice_currency/__manifest__.py:16]: @yweng8111 URL should be `https://github.com/OCA/account-financial-tools`

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

**Pull request closed without merging**